Composer/packages/ui-plugins/selectDialog/src/__tests__/selectDialog.test.tsx

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { cleanup, fireEvent, getAllByRole, render } from 'react-testing-library';
+import { Extension } from '@bfc/extension';
+
+import { SelectDialog } from '../SelectDialog';
+
+const flushPromises = () => new Promise(resolve => setImmediate(resolve));
```

Comment:
I think you can just do something like

```suggestion
const flushPromises = () => Promise.resolve();
```